config.yaml.defaults

```diff
@@ -97,6 +97,9 @@ log:
     # log only those that trigger a warning / error
     api_calls: False
 
+    database_echo: False
```

Comment:
Please add comments explaining what these do.

"echo" is not very descriptive; perhaps use `log.database` and `log.database_pool`?